lib/tasks/dev.rake

```diff
@@ -3,6 +3,33 @@ if Rails.env.development? || Rails.env.test?
   require "factory_bot"
 
   namespace :pomegranate do
+    desc 'Reindex a resource from a IIIF Manifest URL within an Exhibit'
+    task :reindex, %i[manifest exhibit] => [:environment] do |_t, args|
+      manifest = args[:manifest]
+      exhibit_slug = args[:exhibit]
+      exhibit = Spotlight::Exhibit.find_by(slug: exhibit_slug)
+      iiif_resource = IIIFResource.find_or_initialize_by(url: manifest, exhibit: exhibit)
```

Comment:
This will force a manifest into a collection even if it's not in there in Figgy. Maybe just do `find_by`?